tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_import_picker.py

```diff
@@ -26,17 +26,20 @@ def tap_import_button(self, wait_for_import = True):
         # self.marionette.find_element(*self._import_button_locator).tap()
         self.apps.switch_to_displayed_app()
         if wait_for_import:
-            self.wait_for_element_not_displayed(*self._contact_import_picker_frame_locator, timeout=60)
+            Wait(self.marionette, timeout=60).until(expected.element_not_displayed(
+                Wait(self.marionette).until(expected.element_present(
+                    *self._contact_import_picker_frame_locator))))
```

Comment:
This will fail if the element is not present, `expected.element_not_displayed` can take a locator.